client/pages/admin.tsx

```diff
@@ -34,17 +34,23 @@ const Admin: FunctionComponent<AdminProps> = ({ accounts, earlyAccess }) => {
 
 export default Admin
 
-export const getServerSideProps: GetStaticProps = async (context) => {
+export const getServerSideProps: GetServerSideProps = async (context) => {
   let accounts = []
   let earlyAccess = []
   try {
-    let res = await fetch(`${process.env.NEXT_PUBLIC_API_URL_SSR}/admin/accounts`)
-    accounts = await res.json()
-    res = await fetch(`${process.env.NEXT_PUBLIC_API_URL_SSR}/admin/earlyAccess`)
-    earlyAccess = await res.json()
+    let res = await fetch(`${process.env.NEXT_PUBLIC_API_URL_SSR}/admin/accounts`, {
+      headers: context.req ? { cookie: context.req.headers.cookie } : undefined,
```

Comment:
This is how you pass cookies along from Next to the server for its SSR methods. Kind of weird, but it works 🤷‍♂️